tasks.robot

```diff
@@ -56,60 +60,18 @@ Collect Images From User
     ${confidence_threshold} =    Convert To Number    ${confidence_threshold}
     ${similarity_threshold} =    Convert To Number    ${similarity_threshold}
 
-    RETURN    ${result}[query_image]    ${result}[reference_image]
-    ...    ${confidence_threshold}    ${similarity_threshold}
-
-Ask For Retry
-    [Documentation]    Continue or exit the loop of signature verification process.
-
-    Add Submit Buttons    buttons=Retry,Exit    default=Exit
-    &{result} =    Run Dialog    title=${TITLE} - Results
-    ${retry} =    Run Keyword And Return Status    Should Be Equal As Strings
-    ...    ${result}[submit]    Retry
-    RETURN    ${retry}
-
-Display Similar Signatures
-    [Documentation]    Show similar signatures as image crops for manual inspection.
-    [Arguments]    ${qry_path}    ${qry_conf}    ${ref_path}    ${ref_conf}
-    ...    ${status}    ${similarity}
-
-    Clear Dialog
-
-    IF    ${status}
-        Add Icon    Success
-        Add Heading    Signatures match
-    ELSE
-        Add Icon    Warning
-        Add Heading    Signatures don't match
-    END
-
-    Add Heading    Similarity: ${similarity * 100}%
```

Comment:
The similarity score is not displayed anymore, can you add it back?

Reply:
In a different line of code now but fixed.